openmetadata-ui/src/main/resources/ui/src/pages/AlertDetailsPage/AlertDetailsPage.tsx

```diff
@@ -217,6 +220,15 @@ function AlertDetailsPage({
     );
   }, [history]);
 
+  const handleAlertSync = useCallback(async () => {
+    try {
+      await syncOffset(fqn);
+      showSuccessToast(t('label.alert-synced-successfully'));
+    } catch (error) {
+      showErrorToast(error as AxiosError);
+    }
+  }, [fqn]);
```

Comment:
We are not showing any loader while the API is in progress. Can we show a small loader inside the button itself for the same?